hieradata/cluster/ruka/role/rke2server.yaml

```diff
@@ -0,0 +1,9 @@
+---
+classes:
+  - "profile::core::lhnrouting"
```

Comment:
wasn't this getting removed since it was for Centos7?

Reply:
Yes, this branch will need to be rebased after #1403 is merged.